packages/hub/prisma/schema.prisma

```diff
@@ -230,9 +230,12 @@ model ModelRevision {
 model SquiggleSnippet {
   id String @id @default(cuid())
 
-  code    String
-  version String
-  seed    String
+  code          String
+  version       String
+  seed          String
+  autorunMode   Boolean?
+  sampleCount   Int?
+  xyPointLength Int?
```

Comment:
It's probably better (less complex, more explicit) to make these non-optional, and move constants from the frontend to the DB.

But we can do this later.